support/cas-server-support-redis-ticket-registry/src/test/java/org/apereo/cas/ticket/registry/SentinelServerTicketRegistryTests.java

```diff
@@ -0,0 +1,57 @@
+package org.apereo.cas.ticket.registry;
+
+import org.apereo.cas.config.CasCoreTicketCatalogConfiguration;
+import org.apereo.cas.config.CasCoreTicketsConfiguration;
+import org.apereo.cas.config.CasCoreWebConfiguration;
+import org.apereo.cas.config.RedisTicketRegistryConfiguration;
+import org.apereo.cas.config.support.CasWebApplicationServiceFactoryConfiguration;
+import org.apereo.cas.util.junit.EnabledIfContinuousIntegration;
+
+import org.junit.jupiter.api.Tag;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.aop.AopAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.autoconfigure.RefreshAutoConfiguration;
+import org.springframework.context.annotation.EnableAspectJAutoProxy;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+/**
+ * Unit test for {@link RedisTicketRegistry}.
+ *
+ * @author Misagh Moayyed
+ * @since 5.0.0
+ */
+@Tag("Redis")
+@SpringBootTest(classes = {
+    RedisTicketRegistryConfiguration.class,
+    RefreshAutoConfiguration.class,
+    CasCoreWebConfiguration.class,
+    AopAutoConfiguration.class,
+    CasCoreTicketsConfiguration.class,
+    CasCoreTicketCatalogConfiguration.class,
+    CasWebApplicationServiceFactoryConfiguration.class})
+@TestPropertySource(properties = {
+    "cas.ticket.registry.redis.host=localhost",
+    "cas.ticket.registry.redis.port=6379",
+    "cas.ticket.registry.redis.pool.max-active=20",
+    "cas.ticket.registry.redis.sentinel.master=mymaster",
+    "cas.ticket.registry.redis.sentinel.node[0]=localhost:26379",
+    "cas.ticket.registry.redis.sentinel.node[1]=localhost:26380",
+    "cas.ticket.registry.redis.sentinel.node[2]=localhost:26381"
+})
+@EnableTransactionManagement(proxyTargetClass = true)
+@EnableAspectJAutoProxy(proxyTargetClass = true)
+@EnabledIfContinuousIntegration
+public class SentinelServerTicketRegistryTests extends BaseTicketRegistryTests {
```

Comment:
SentinelServerTicketRegistryTests seems identical to ServiceRegistryTests run on CI. Perhaps duplications can be refactored to a common redis module and reused here?

Reply:
Also the last commit purpose a refactored way